parlai/scripts/train_model.py

```diff
@@ -44,7 +44,7 @@ def setup_args(parser=None):
     train.add_argument('-et', '--evaltask',
```

Comment:
strip these changes.

Reply:
+1. None of these are necessary for the metadialog project.